gradle/dependencies.gradle

```diff
@@ -1,9 +1,15 @@
 ext.deps = [
     kotlin: [
-        stdlib: "org.jetbrains.kotlin:kotlin-stdlib",
+        stdlibcommon: "org.jetbrains.kotlin:kotlin-stdlib-common",
+        stdlibjdk8: "org.jetbrains.kotlin:kotlin-stdlib-jdk8",
```

Comment:
What JDK8 specific features of the stdlib are we using?

Reply:
This include was part of the boilerplate of [Setting up a Multiplatform Project](https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html#setting-up-a-multiplatform-project) and I haven't yet revisited to ensure all of the configuration was necessary. No jdk8 features have been added, the dependency will be removed.